helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -138,6 +138,42 @@ k8sattributes:
     {{- end }}
 {{- end }}
 
+{{/*
+Common config for K8s attributes processor adding k8s metadata to metrics resource attributes.
+*/}}
+{{- define "splunk-otel-collector.k8sAttributesProcessorMetrics" -}}
+k8sattributes/metrics:
+  pod_association:
+    - sources:
+      - from: resource_attribute
+        name: k8s.node.name
+    - sources:
+      - from: resource_attribute
+        name: k8s.pod.uid
+    - sources:
+      - from: resource_attribute
+        name: k8s.pod.ip
+    - sources:
+      - from: resource_attribute
+        name: k8s.namespace.name
+    - sources:
+      - from: resource_attribute
+        name: ip
+    - sources:
+      - from: connection
+    - sources:
+      - from: resource_attribute
+        name: host.name
+  extract:
+    annotations:
+      - key: splunk.com/metricsIndex
+        tag_name: com.splunk.index
+        from: namespace
+      - key: splunk.com/metricsIndex
+        tag_name: com.splunk.index
+        from: pod
```

Comment:
It would be great if we can also add `sourcetype` extraction like https://github.com/signalfx/splunk-otel-collector-chart/blob/main/helm-charts/splunk-otel-collector/templates/config/_common.tpl#L117-L118 for metrics.